zsh-aws-vault.plugin.zsh

```diff
@@ -13,6 +13,35 @@ function avsh() {
   aws-vault exec $1 -- zsh
 }
 
+function aws-login() {
+    aws-vault login $1
+}
+
+function aws-login-incognito() {
```

Comment:
Nice - this is useful. To match the existing convention, maybe this could be `avli`?